_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1850,21 +1884,61 @@ include {
-Finally, deploy the `account-baseline` module by running `terragrunt apply`:
+You're now going to use an IAM role to authenticate to the logs account. This IAM role is created automatically in each
+child account by `account-baseline-root` and has a default name of `OrganizationAccountAccessRole`. There are many ways
+to https://blog.gruntwork.io/a-comprehensive-guide-to-authenticating-to-aws-on-the-command-line-63656a686799[assume an IAM role on the CLI];
+for this guide, we're going to keep using `aws-vault`.
+
+Open up `~/.aws/config` and you should see a `profile` that was created automatically when you ran
+`aws-vault add root-iam-user`  earlier:
+
+[source,text]
+----
+[profile root-iam-user]
+----
+
+Add a new `profile` entry in `~/.aws/config` for your logs account that uses the `root-iam-user` as the
+`source_profile`:
+
+[source,text]
+----
+[profile logs-from-root]
+role_arn=arn:aws:iam::<LOGS_ACCOUNT_ID>:role/OrganizationAccountAccessRole
+source_profile=root-iam-user
+----
+
+Check that you're able to authenticate to the logs account:
+
+[source,bash]
+----
+aws-vault exec logs-from-root -- aws sts get-caller-identity
+----
+
+You should see JSON output indicating that you've successfully assumed an IAM role:
+
+[source,json]
+----
+{
+  "UserId": "AIDAXXXXXXXXXXXX:1597932316055520000",
+  "Account": "<LOGS_ACCOUNT_ID>",
+  "Arn": "arn:aws:sts::<LOGS_ACCOUNT_ID>:assumed-role/OrganizationAccountAccessRole/1597932316055520000"
+}
+----
+
+You're now ready to deploy the `account-baseline` module in the logs account by running `terragrunt apply`:
 
 [source,bash]
 ----
 cd infrastructure-live/logs/_global/account-baseline
-terragrunt apply -parallelism=4
+aws-vault exec logs-from-root -- terragrunt apply -parallelism=2
 ----
 
-IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=4`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors. On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid `Error: pipe: too many open files` errors by running: `ulimit -n 1024`.
+IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=2`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors. On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid `Error: pipe: too many open files` errors by running: `ulimit -n 1024`.
```

Comment:
Same comment as above re: `ulimit`.

Reply:
Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/1d9b5b79c84044d7f8fb6225df4379a2399bdfe8 and https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/c6a37ff8c87fd62b63173a56e3b286cff2f81b8b.